src/control/cmd/dmg/pool.go

```diff
@@ -188,7 +189,47 @@ func (d *PoolDestroyCmd) Execute(args []string) error {
 	return err
 }
 
-// PoolQueryCmd is the struct representing the command to destroy a DAOS pool.
+// PoolReintegrateCmd is the struct representing the command to Add a DAOS target.
+type PoolReintegrateCmd struct {
+	logCmd
+	connectedCmd
+	UUID    string `long:"pool" required:"1" description:"UUID of the DAOS pool to start reintegration in"`
+	Rank    uint32 `long:"rank" required:"1" description:"Rank of the targets to be reintegrated"`
+	Targets string `long:"idx" required:"1" description:"list of idx(s) to be reintegrated into the rank"`
+}
+
+// Execute is run when PoolReintegrateCmd subcommand is activated
+func (r *PoolReintegrateCmd) Execute(args []string) error {
+	msg := "succeeded"
+
+	targetlist := make([]uint32, 0)
+	if len(r.Targets) > 0 {
+		targetStr := strings.Split(r.Targets, ",")
+		for _, rank := range targetStr {
+			t, err := strconv.Atoi(rank)
+			if err != nil {
+				return errors.WithMessage(err, "parsing rank list")
+			}
+			if t < 0 {
+				return errors.Errorf("invalid rank: %d", t)
+			}
+			targetlist = append(targetlist, uint32(t))
+		}
+	}
```

Comment:
Worth noting, @tanabarr just landed a PR to master which adds a `common.ParseInts()` helper that can be used instead of this. We're also going to improve and generalize it to `common.ParseNumberList()` in a follow-on PR so that the helper can parse any sequence of valid numbers.

Reply:
I did see @tanabarr's commit and referenced it in a comment above. I wanted to get the PR started and wasn't sure how long it would take to land. 

Now that it's landed I'll rebase my patch and use that instead.